src/components/Footer.tsx

```diff
@@ -42,7 +42,7 @@ const Footer = () => (
               isExternal
               href="https://vercel.com/?utm_source=octoclairvoyant-team&utm_campaign=oss"
             >
-              <Image src={poweredByVercelLogo} />
+              <Image alt="Powered by Vercel logo" src={poweredByVercelLogo} />
```

Comment:
We have [`jsx-a11y/alt-text`](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/alt-text.md) ESLint rule enabled, but didn't report the error since we are using a custom `Image` component for rendering the `img` tag. Can you generate a ticket for me to set up that rule properly so we catch more similar problems in the future?

Reply:
See issue #227. 😉